src/api/v1/community_businesses/__tests__/visit_activites.test.integration.ts

```diff
@@ -0,0 +1,152 @@
+            monday: false,
+            tuesday: false,
+            wednesday: true,
+            thursday: false,
+            friday: false,
+            saturday: false,
+            sunday: false,
+          }),
+          expect.objectContaining({
+            id: 3,
+            name: 'Free Running',
+            category: 'Sports',
+            monday: false,
+            tuesday: true,
+            wednesday: true,
+            thursday: false,
+            friday: true,
+            saturday: true,
+            sunday: true,
+          }),
+        ]),
+      });
+    });
+  });
+  describe('PUT', () => {
+    test(':: successfully updates a visit activity', async () => {
+      const res = await (<any> server).inject({
+        method: 'PUT',
+        url: '/v1/community-businesses/me/visit-activities/1',
+        payload: {
+          id: 2,
+          monday: true,
+        },
+        credentials: {
+          user: 1 ,
+          organisation: 1 ,
+          scope: ['visit_activities-own:write'],
+        },
+      });
+
+      expect(res.statusCode).toBe(200);
+      expect(res.result).toEqual(
+        { result: expect.objectContaining(
+          {
+            id: 2,
+            name: 'Wear Pink',
+            monday: true,
+            tuesday: false,
+            wednesday: true,
+            thursday: false,
+            friday: false,
+            saturday: false,
+            sunday: false,
+          }),
+        });
+    });
+  });
+
+  describe('POST', () => {
+    test(':: successfully add a new activity', async () => {
+      const res = await (<any> server).inject({
+        method: 'POST',
+        url: '/v1/community-businesses/me/visit-activities',
+        payload: {
+          name: 'Base Jumping',
+          category: 'Adult skills building',
+        },
+        credentials: {
+          user: 1 ,
+          organisation: 1 ,
+          scope: ['visit_activities-own:write'],
+        },
+      });
+
+      expect(res.statusCode).toBe(200);
+      expect(res.result).toEqual(
+        { result: expect.objectContaining(
+          {
+            id: 5,
+            name: 'Base Jumping',
+            monday: false,
+            tuesday: false,
+            wednesday: false,
+            thursday: false,
+            friday: false,
+            saturday: false,
+            sunday: false,
+          }),
+        });
+    });
+  });
+
+  describe('DELETE', () => {
+    test(':: successfully deletes an activity', async () => {
+      const res = await (<any> server).inject({
+        method: 'DELETE',
+        url: '/v1/community-businesses/me/visit-activities/1',
+        credentials: {
+          user: 1 ,
+          organisation: 1 ,
+          scope: ['visit_activities-own:delete'],
+        },
+      });
+
+      expect(res.statusCode).toBe(200);
+      expect((<any> res.result).result.deletedAt).toBeTruthy();
+      const doubleCheck = await CommunityBusinesses.getVisitActivities(server.app.knex, { id: 1 });
```

Comment:
This is fine, but I actually meant to make a new `GET` request and check the response. The latter is more decoupled from the implementation of the handler logic, and operates on the same level of abstraction as the functionality under test (i.e. you're testing from the perspective of a consumer of the HTTP API, who wouldn't have access to lower level abstractions -- in this case the models).

This is just an FYI though, doesn't need to be done.